### PR TITLE
[SDK-4499] Update Next.js QS for app dir

### PR DESCRIPTION
**Pull request by adamjmcgrath**
image

### CODE REVIEW

File: articles/quickstart/webapp/nextjs/01-login.md [hunk 1]
```diff
@@ -164,6 +166,28 @@ The `user` property contains sensitive information and artifacts related to the
 - Ensure that the SDK has loaded successfully by checking that no `error` was produced.
 - Check the `user` property to ensure that Auth0 has authenticated the user before React renders any component that consumes it.
 
+### From a Client Component
```
Comment (frederikprijck) [inline articles/quickstart/webapp/nextjs/01-login.md]:
```suggestion
### From a Server Component
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: articles/quickstart/webapp/nextjs/01-login.md [hunk 2]
```diff
@@ -164,6 +166,28 @@ The `user` property contains sensitive information and artifacts related to the
...
+          <div>
+            <img src={user.picture} alt={user.name} />
+            <h2>{user.name}</h2>
+            <p>{user.email}</p>
+          </div>
+      )
+  );
+}
+```
+
```
Comment (Widcket) [inline articles/quickstart/webapp/nextjs/01-login.md:190]: Should we include a panel with a link to https://github.com/auth0/nextjs-auth0#using-this-sdk-with-react-server-components?

File: articles/quickstart/webapp/nextjs/interactive.md [hunk 3]
```diff
@@ -155,3 +148,21 @@ Sorry about that. Here's a couple of things to double check:
...
+:::
+:::checkpoint-failure
+Sorry about that. Here's a couple of things to double check:
+* are your environment variables populated correctly?
+* make sure you have successfully logged in through the `/api/auth/login` handler.
+* make sure there are no errors in the console
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+:::
+::::
```
Comment (Widcket) [inline articles/quickstart/webapp/nextjs/interactive.md:168]: Same as above.

File: articles/quickstart/webapp/nextjs/01-login.md (hunk 2 above)
Comment (adamjmcgrath) [reply articles/quickstart/webapp/nextjs/01-login.md:190]: I don't think that's necessary and complicates the quickstart - they'll get the warning in the console.

**Review by Widcket — APPROVED** [review]

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**